app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const {getTopics} = require('./controllers/ncnews.controllers')
+const app = express()
+
+app.get('/api/topics', getTopics)
+
+app.all('*', (req, res) =>{
```

Comment:
Awesome to see the app.all here to send back a custom 404, your test for this is perfect!